docs/REST.md

```diff
@@ -24,6 +25,11 @@ When performing GET on `<host>/repair-scheduler/v1/list` a [JSON list of RepairJ
 When performing GET on `<host>/repair-scheduler/v1/list/mykeyspace` a JSON list of RepairJobs for that specific keyspace will be returned.
 
 
+### List repair config
```

Comment:
Might be a small difference here between the Karaf command and the REST request in how well we are able to push large amounts of data. Would it make sense to add some kind of sever-side limitation to how much data is fetched as well? (E.g. adding keyspace and possible table to show config for)

Reply:
Added the possibility to limit by keyspace (similar as list).
Made a change for this in a separate commit...